dev/create-release/generate-contributors.py

```diff
@@ -88,7 +88,7 @@ def print_indented(_list):
 
 
 def is_release(commit_title):
-    return re.findall("\[release\]", commit_title.lower()) or \
+    return re.findall(r"\[release\]", commit_title.lower()) or \
```

Comment:
1. Could we use parents to remove line terminating backslashes as recommended in PEP8?
2. Could we get rid of the use of __re__ in this instance with
```python
    return ("[release]", commit_title.lower() or
         "preparing spark release" in commit_title.lower() or
         "preparing development version" in commit_title.lower() or
         "CHANGES.txt" in commit_title)
 ```